mythical-creatures/exercises/hobbit.js

```diff
@@ -0,0 +1,22 @@
+function Hobbit(name, disposition, age, adult, isShort, old){
+  this.name = name;
+  this.disposition = disposition || 'homebody';
+  this.age = age || 0;
+  this.adult = () => {
+    if(this.age >= 33){
```

Comment:
But doesn't that still make `this.adult` a method rather than a property? If you call `hobbit.adult` does it return true/false? I'll play with it tonight to see what the tests are telling me.